app/views/admin/overview/index.html.erb

```diff
@@ -0,0 +1,54 @@
+<h2 class="crayons-title mb-3">Overview</h2>
+<div class="grid s:grid-cols-3 s:grid-rows-4 gap-3">
+  <article style="grid-column: span 2; grid-row: span 4;">
+    <section class="crayons-card mb-3">
+      <div class="crayons-card__body" data-controller="ahoy">
+        <h3 class="crayons-subtitle-2">Welcome Checklist</h3>
+        <p class="crayons-field__description mb-3">Here are things we recommend you do now that you're on Forem</p>
+        <ol>
+          <li>Setup and configure your Forem under <a href="/admin/config" data-action="click->ahoy#trackOverviewLink">Config</a></li>
+          <li><a href="/admin/invitations" data-action="click->ahoy#trackOverviewLink">Invite people</a> to join your Forem</li>
+          <li><a href="/admin/tags" data-action="click->ahoy#trackOverviewLink">Update your Forem tags</a></li>
+          <li><a href="/admin/pages" data-action="click->ahoy#trackOverviewLink">Create pages</a> and <a href="/admin/navigation_links" data-action="click->ahoy#trackOverviewLink">sort them in the navigation sidebar</a></li>
+          <li>Read the <a href="https://forem.gitbook.io/forem-admin-guide/quick-start-guide" data-action="click->ahoy#trackOverviewLink">Quick Start Guide</a></li>
+          <li><a href="/admin/welcome" data-action="click->ahoy#trackOverviewLink">Set up a Welcome Thread</a></li>
+          <li>Browse the <a href="https://forem.gitbook.io/forem-admin-guide" data-action="click->ahoy#trackOverviewLink">Forem Admin Guide</a></li>
+          <li>Join the private <a href="https://creators.forem.com">Creators Forem</a>! (email <a href="mailto:support@forem.com" data-action="click->ahoy#trackOverviewLink">support@forem.com</a> if you need an invite)</li>
```

Comment:
```suggestion
          <li>Join the private <a href="https://creators.forem.com">Creators Forem</a> (email <a href="mailto:support@forem.com" data-action="click->ahoy#trackOverviewLink">support@forem.com</a> if you need an invite)</li>
```
Just to keep it consistent with the other items in the list don't have punctuation :)